### PR TITLE
use cluster pool for cincinnati-graph-data tests

### PR DESCRIPTION
**Pull request by PratikMahajan**

### CODE REVIEW

**Comment by vrutkovs** [comment]
/retest

This also needs another `make update` to move `optional` to expected position

File: ci-operator/config/openshift/cincinnati-graph-data/openshift-cincinnati-graph-data-master.yaml [hunk 1]
```diff
@@ -48,12 +48,12 @@ raw_steps:
 releases:
   initial:
     integration:
-      name: "4.11"
+      name: "4.10"
```
Comment (wking) [inline ci-operator/config/openshift/cincinnati-graph-data/openshift-cincinnati-graph-data-master.yaml:51]: I don't understand this shift from 4.11 to 4.10.  Can you explain your motivation?

Comment (PratikMahajan) [reply ci-operator/config/openshift/cincinnati-graph-data/openshift-cincinnati-graph-data-master.yaml:51]: the cluster pools dont have a 4.11 cluster as per a previous job run. so decided to move it to 4.10
here's the job https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/26913/rehearse-26913-pull-ci-openshift-cincinnati-graph-data-master-e2e-latest-cincinnati/1502035638516453376

Comment (wking) [reply ci-operator/config/openshift/cincinnati-graph-data/openshift-cincinnati-graph-data-master.yaml:51]: makes sense.  Can we add that motivation to the commit message?

**Pull request merged**